lib/ffmpeg/transcoderbase.py

```diff
@@ -165,8 +169,8 @@ def gen_output_opts(self):
         filters.extend(["hwdownload", f"format={self.map_format(format)}"])
       elif ("sw", "hw") == tmode: # SW to HW transcode
         filters.extend([f"format={self.map_format(format)}", "hwupload"])
-        if vars(self).get("hwframes", None) is not None:
-          filters[-1] += "=extra_hw_frames={hwframes}"
+        if vars(self).get("ehwframes", None) is not None:
```

Comment:
How about use var name `ohwframes`, instead, since the function name refers to "output" opts?